tests/test_reader.py

```diff
@@ -1290,8 +1290,8 @@ def test_reader(caplog):
     caplog.clear()
     # first call requires some reparations...
     reader.pages[0].extract_text()
-    assert "repaired" in caplog.text
-    assert "found" in caplog.text
+    # assert "repaired" in caplog.text
```

Comment:
Is this really a side effect of the single changed line which allows disabling this check?

Reply:
yes it is the rebuilt process removes the warnings